opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchSource.java

```diff
@@ -273,10 +291,13 @@ public SourceResponse query(QueryRequest queryRequest) throws UnsupportedQueryEx
                 Metacard metacard = null;
                 List<Result> resultQueue = new ArrayList<Result>();
                 try {
-                    metacard = inputTransformer.transform(responseStream);
-                } catch (IOException e) {
-                    LOGGER.debug("Problem with transformation.", e);
-                } catch (CatalogTransformerException e) {
+                    final byte[] bytes = IOUtils.toByteArray(responseStream);
```

Comment:
We could use a system property and then set it in the system.properties file in karaf.home/etc.

http://docs.oracle.com/javase/7/docs/api/java/lang/System.html#getProperty(java.lang.String,%20java.lang.String)